### DISCUSSION TITLE
Azure ACI - should I set SEQ_FEATURES_ENABLED nativemetastore

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [datalust/seq-tickets#2161] [Help] [answered]
According to discussions responses to this page https://ml-software.ch/posts/running-seq-on-azure it is advisible to set the following property when deploying Seq to Azure container instance.  Can someone confirm if this is desirable and still required?  If so, should it be added to the ACI instructions? 

        - name: SEQ_FEATURES_ENABLED
          value: nativemetastore

**Comment by liammclennan** [comment, marked as answer]
Hi @rodneytamblyn 

Native meta store is now the default, so that setting is no longer required and the docs are correct.
